projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -662,6 +662,11 @@ export interface FeatureTogglesInterface {
    */
   a11yDeleteButton2First?: boolean;
 
+  /**
+   * Show label of every `ng-select` and `select`.
```

Comment:
Let's mention what components we have changed. We actually have some dos and don'ts written down that can help: 
[2.18.2. Write a JSDoc description of a Feature Toggle](https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-WriteaJSDocdescriptionofaFeatureToggle)

Reply:
I've just updated it.